### ISSUE TITLE
Variable initialization in try block causes compiler error

### ISSUE THREAD

**Issue by ykoitzsch** [neow3j/neow3j#547]
I realised that initializing a variable within the try block will cause a compiler error.

This works:

```
public static String test() {
        String foo;
        try {
            foo = "hello world";
        } catch (Exception e) {}
        return "test";
    }
```

This doesn't:
```
public static String test() {
        try {
            String hello = "world";
        } catch (Exception e) {

        }
        return "foo";
    }
```
Exception:
```
Exception in thread "main" io.neow3j.compiler.CompilerException: The method 'test' from io.neow3j.examples.contractdevelopment.contracts.TryCatchTest was not compiled with debugging information and can therefore not be used for smart contract compilation. Make sure to only use methods from your workspace or smart contract libraries.
        at io.neow3j.compiler.NeoMethod.checkForMissingLocalVariableInformation(NeoMethod.java:757)
        at io.neow3j.compiler.NeoMethod.initializeLocalVariablesAndParameters(NeoMethod.java:730)
        at io.neow3j.compiler.NeoMethod.initialize(NeoMethod.java:725)
        at io.neow3j.compiler.Compiler.initializeContractMethods(Compiler.java:373)
        at io.neow3j.compiler.Compiler.compile(Compiler.java:266)
        at io.neow3j.compiler.Compiler.compile(Compiler.java:240)
        at io.neow3j.examples.contractdevelopment.CompileAndDeploy.main(CompileAndDeploy.java:29)
```

**Comment by gsmachado** [comment]
Hey @ykoitzsch

We really appreciate your reports! 💪 

As I mentioned [in this comment](https://github.com/neow3j/neow3j/issues/546#issuecomment-841807445), can you try to compile using the `3.10.0-SNAPSHOT`?

Then, please, give us feedback on whether the problem was fixed or not.

**Comment by gsmachado** [comment]
@ykoitzsch ok, I could confirm that we still have this problem (current `develop-3.x` branch). I've made some tests demonstrating the problem (that at the moment are failing). We'll fix asap.

@csmuller, just FYI.

**Comment by ykoitzsch** [comment]
@gsmachado great, I just didn't have the time yet to test it myself in the latest snapshot version. I'll keep that in mind for the next time :)

**Comment by csmuller** [comment]
Fixed in #548.
Will be release in 3.10.0 this week.
